README.md

```diff
@@ -27,10 +27,10 @@ If you are interested in contributing to NATS, read about our...
 [License-Image]: https://img.shields.io/badge/License-Apache2-blue.svg
 [Fossa-Url]: https://app.fossa.io/projects/git%2Bgithub.com%2Fnats-io%2Fgnatsd?ref=badge_shield
 [Fossa-Image]: https://app.fossa.io/api/projects/git%2Bgithub.com%2Fnats-io%2Fgnatsd.svg?type=shield
-[Build-Status-Url]: https://travis-ci.org/nats-io/nats-server
-[Build-Status-Image]: https://travis-ci.org/nats-io/nats-server.svg?branch=master
-[Release-Url]: https://github.com/nats-io/nats-server/releases/tag/v2.1.8
-[Release-image]: https://img.shields.io/badge/release-v2.1.8-1eb0fc.svg
+[Build-Status-Url]: https://travis-ci.com/github/nats-io/nats-server
+[Build-Status-Image]: https://travis-ci.com/nats-io/nats-server.svg?branch=master
```

Comment:
Should this just be for this branch and not master?

Reply:
Yes, but anybody can technically select the branch in the UI and will have whatever is displayed in the README. So why not update there too.

Reply:
Oh, I see what you mean..

Reply:
will update all references from master to `branch_2_1_x` to reflect actual values.

Reply:
I pushed the update.